src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -67,8 +68,8 @@ private String getCloudInputs() {
         final String localResources =  getResourceDir();
         final String hg19Ref = ReferenceAPISource.HG19_REF_ID;
         final String GRCh37Ref = ReferenceAPISource.URL_PREFIX + ReferenceAPISource.GRCH37_REF_ID;
-        final String HiSeqBam_chr20 = localResources + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam";
-        final String dbSNPb37_chr20 = localResources + "dbsnp_138.b37.excluding_sites_after_129.ch20.1m-1m1k.vcf";
+        final String HiSeqBam_chr20 = localResources + BaseRecalibratorIntegrationTest.WGS_B37_CH20_1M_1M1K_BAM;
+        final String dbSNPb37_chr20 = localResources + BaseRecalibratorIntegrationTest.DBSNP_138_B37_CH20_1M_1M1K_VCF;
```

Comment:
These should live as constants in `BaseTest`, not `BaseRecalibratorIntegrationTest` -- it's gross for one test class to access something from another like this.